### ISSUE TITLE
[Table of Content]: QA: Unable to navigate through all contents with Voice over

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9423]
### Description

Page Element:
Table of Content

Issue Description:
Unable to navigate through all the sections in TOC using left/Right swipe in iPhone Voiceover. When we try to navigate using Left/Right swipe, the screen reader focus gets stuck between Two sections.

Expected:
User is expected to navigate through all the content using Left/Right swipe.

Impact:
When the user is unable to swipe through the page elements, it impacts the ability to navigate in the page. This affects the mobile screen reader user.

### Component(s) impacted

Table of content-Web components-Canary/Staging

### Browser

Safari

### Carbon for IBM.com version

v1.39.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component Staging- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
